lib/matplotlib/axis.py

```diff
@@ -988,19 +1051,27 @@ def _translate_tick_params(kw):
             'labelright': 'label2On',
             'labeltop': 'label2On',
```

Comment:
labelleft & labelbottom have the same axis key, the translation will therefore always return `labelleft` and `labelright`, also for the x-axis. This is confusing.

Reply:
Please open a new issue with the details of what you are seeing.

Reply:
Ok, done, see #28772